cloudbuild.yml

```diff
@@ -1,13 +1,16 @@
 steps:
 - id: 'install'
-  name: 'gcr.io/cloud-builders/npm'
+  name: 'node:19.0.0'
```

Comment:
Do we have to pin it all the way? There is no 19.0.x option?

Reply:
Not necessarily, but I believe the `gcr.io/cloud-builders/npm`uses `19.0.0` right now, so pinning to that version should have the least regression

Reply:
Mkay. Thinking patch versions should be pretty safe though? But this is good I'm sure :+1: